src/confidential-accounts/confidential-accounts.service.ts

```diff
@@ -152,4 +153,13 @@ export class ConfidentialAccountsService {
 
     return account.getTransactionHistory(filters);
   }
+
+  public async moveFunds(params: MoveFundsDto): ServiceReturn<void> {
```

Comment:
This method should actually generate proof using the proof server and then submit to SDK. Check other methods where proof is being generated for reference